.github/workflows/ci.yml

```diff
@@ -9,7 +9,7 @@ on:
 env:
   RUST_BACKTRACE: 1
   RUSTFLAGS: -Dwarnings
-  MSRV: "1.63"
+  MSRV: "1.65"
```

Comment:
This should be OK for deltachat.

https://github.com/deltachat/deltachat-core-rust/blob/master/CHANGELOG.md#11130---2023-04-18